src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesTaskLauncherProperties.java

```diff
@@ -35,6 +35,11 @@ public class KubernetesTaskLauncherProperties {
 	 */
 	private Integer backoffLimit;
 
+	/**
+	 * The time to life value to limit the lifetime of Jobs that have finished execution.
```

Comment:
Do you mind updating this slightly to include the caveat that removed jobs will not be able to have their logs retrieved. Something like this:

```
	 * The number of seconds after a job has finished before it is eligible to be automatically removed by the TTL 
	 * controller - note that logs from removed jobs will not be able to be retrieved.
```

Reply:
Also, you have a checkstyle failure    https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/runs/7437762079?check_suite_focus=true#step:6:990

Reply:
Thanks for Your comments @onobc. I'd change the comment and fix the checkstyle issue.